client/components/LearnerViewDirective.js

```diff
@@ -24,24 +24,41 @@ tie.directive('learnerView', [function() {
       <h3>Exercise: {{title}}</h3>
 
       <div class="tie-learner-view-left-column">
+        <div class="tie-old-instructions">
```

Comment:
Oh no wait I got it. Might be worth a comment, or calling them "previous-instructions"?